### PR TITLE
Add how-to/migrate-pki-to-sha256.html.md

### PR DESCRIPTION
**Pull request by didib**
Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

File: source/documentation/how-to/migrate-pki-to-sha256.html.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+---
+title: Migrate PKI to SHA256 signatures Howto
+category: howto
+---
+
+# Migrate PKI to SHA256 signatures Howto
+
+## Summary
+
+oVirt 4.1, on new setups, creates PKI that uses SHA256 signatures.
```
Comment (mykaul) [inline source/documentation/how-to/migrate-pki-to-sha256.html.md]: PKI infrastructure

File: source/documentation/how-to/migrate-pki-to-sha256.html.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+Existing setups upgraded to 4.1 do not currently have PKI migrated.
+
+This Howto explains how to manually migrate the PKI of such setups
+to use SHA256 signatures.
+
+## Background
+
+Previous versions of oVirt used SHA-1 for signatures of ssl certificates
```
Comment (mykaul) [inline source/documentation/how-to/migrate-pki-to-sha256.html.md]: ssl -> SSL

File: source/documentation/how-to/migrate-pki-to-sha256.html.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+You might need to restart it and/or remove the engine cert and/or engine ca cert.
+
+In my own case I unchecked "Permanently store this exception" when I first
+logged in, and after restarting httpd the browser showed an error about using
+the same serial number. Restarting the browser was enough to login again.
+
+## Re-add hosts
+
+For all of your hosts, one host at a time, using the web admin ui:
+* Set it to Maintenance
```
Comment (mykaul) [inline source/documentation/how-to/migrate-pki-to-sha256.html.md]: How can you set it to maint.? If you've recreated the CA, how does it communicate with the hosts?

Comment (didib) [inline source/documentation/how-to/migrate-pki-to-sha256.html.md]: That's a good question that I thought will be a problem and didn't investigate yet. @mwperina - can you help please? However, it very much did work. I also verified migration between a host with sha1 cert and a host with sha256, in both directions.

File: source/documentation/how-to/migrate-pki-to-sha256.html.md (hunk 1 above)
Comment (didib) [inline source/documentation/how-to/migrate-pki-to-sha256.html.md]: Done

File: source/documentation/how-to/migrate-pki-to-sha256.html.md (hunk 2 above)
Comment (didib) [inline source/documentation/how-to/migrate-pki-to-sha256.html.md]: Done

File: source/documentation/how-to/migrate-pki-to-sha256.html.md (hunk 3 above)
Comment (didib) [inline source/documentation/how-to/migrate-pki-to-sha256.html.md]: BTW, please note that I am not re_creating_ the CA - I keep the same key. I just sign it again. I am not an expert in this, so not sure if that's a complete explanation.

Comment (didib) [inline source/documentation/how-to/migrate-pki-to-sha256.html.md]: I'll change the text for now.

Comment (didib) [inline source/documentation/how-to/migrate-pki-to-sha256.html.md]: Changed to 're-sign'.

**Pull request merged**